apps/frontend/src/libs/components/nav-item/nav-item.tsx

```diff
@@ -0,0 +1,31 @@
+import { NavLink } from "~/libs/components/components.js";
+import { IconMap } from "~/libs/enums/enums.js";
+import { combineClassNames } from "~/libs/helpers/combine-class-names.helper.js";
+import { useCallback } from "~/libs/hooks/hooks.js";
+import { type NavigationItem } from "~/libs/types/types.js";
+
+import styles from "./styles.module.css";
+
+const NavItem = ({ href, icon, label }: NavigationItem): React.JSX.Element => {
+	const Icon = IconMap[icon];
+
+	const getNavLinkClassName = useCallback(
+		({ isActive }: { isActive: boolean }): string =>
+			combineClassNames(
+				styles["sidebar-navigation-link"],
+				isActive && styles["sidebar-navigation-link-active"],
+			),
+		[],
+	);
```

Comment:
let's simplify classnames a bit, e.g., icon, label. I remember I was mentioning it once in chat, that since we are using css modules and relatively small components, we need to give meaningful names to classes, but we don't need to clarify the context, because we already see it from the component name and uniqueness is guaranteed
```suggestion
	const getNavLinkClassName = useCallback(
		({ isActive }: { isActive: boolean }): string =>
			combineClassNames(
				styles["link"],
				isActive && styles["link-active"],
			),
		[],
	);
```